packages/browser/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Version 0.3.1
```

Comment:
Maybe it would be a good idea to sync this version to sdk-core version?

Reply:
I disagree, there will be a situation when a specific SDK will be updated without sdk-core update. I prefer to leave it as it is and continue updating libraries this way.